doc/api/dgram.md

```diff
@@ -226,6 +241,24 @@ never have reason to call this.
 If `multicastInterface` is not specified, the operating system will attempt to
 drop membership on all valid interfaces.
 
+### socket.dropSourceSpecificMembership(sourceAddress, groupAddress[, multicastInterface])
+<!-- YAML
+added: vX.Y.Z
```

Comment:
It's already pointed out above, but `added: REPLACEME` here as well. The reason is so that it will be properly detected and replaced during the release process.

Reply:
fixed in 305693a